src/usecases/retrieveVisits.contractTest.js

```diff
@@ -0,0 +1,76 @@
+import AppContainer from "../containers/AppContainer";
+import moment from "moment";
+import deleteVisitByCallId from "./deleteVisitByCallId";
+
+describe("retrieveVisits contract tests", () => {
+  const container = AppContainer.getInstance();
+
+  it("retrieves all Trusts from the db", async () => {
```

Comment:
Should that be 'visits' instead of 'trusts'?

Reply:
copy paste fail, nice catch